src/systems/filecoin_markets/storage_market/_index.md

```diff
@@ -3,17 +3,115 @@ menuTitle: Storage Market
+3. The `StorageClient` now creates a `StorageDealProposal` and sends the proposal and the CID for the root of the data to be stored to the `StorageProvider` using the `Storage Deal Protocol`
+
+Execution now moves to the `StorageProvider`
+
+4. The `StorageProvider` inspects the deal to verify that its parameters match it's own internal criteria (such as price, piece size, deal duration, etc) -- and rejects the proposal if it doesn't
+5. The `StorageProvider` queries the `StorageMarketActor` to verify the `StorageClient` has deposited enough funds to make the deal (i.e. balance > total storage price) and rejects the proposal if it hasn't.
+6. If all criteria are met, the `StorageProvider` responds on the `Storage Deal Protocol` indicate an intent to accept the deal.
+
+Execution now moves back to the `StorageClient`
+
+7. The `StorageClient` opens a push request using the `Data Transfer Module`, and sends the request to the provider along with a voucher containing the CID for the `StorageDealProposal` 
+8. The `StorageProvider` checks the voucher and verifies the CID matches an open proposal, and, if so, accepts the data transfer request from the `StorageClient`.
+9. The `Data Transfer Module` now transfers for the data to be stored from the  `StorageClient` to the `StorageProvider` using `GraphSync`.
+10. Once complete, the `Data Transfer Module` notifies the `StorageProvider`
+11. The `StorageProvider` recalculates the piece commitment (`CommP`) from the data that was transferred and verifies it matches the piece commitment in the the `StorageDealProposal`
+
+# Publishing
+
+Data is now transferred, both parties have agreed, and it's time for the `StorageProvider` to publish the deal.
+
+12. First, the `StorageProvider` add collateral for the deal as needed to the `StorageMarketActor` (using `AddBalance`)
+13. Now, the `StorageProvider` calls `PublishStorageDeals` on the `StorageMarketActor` to publish the deal. It sends the StorageDealProposal signed by the client as well as its own signature in the message to publish the storage deal.
+14. If all goes well, the `StorageMarketActor` responds with an on chain `DealID` for the published deal.
+15. The `StorageProvider` responds sends a message to the `StorageClient` on the `Storage Deal Protocol` with the on chain `DealID` to indicate the deal has been accepted and published.
+
+Finally, the client verifies the deal.
+
+16. The `StorageClient` should query the `StorageMarketActor` with the `DealID` returned by the `StorageProvider` to get the published `StorageDeal` signed by the `StorageProvider`
+
+# Handoff
+
+Now that a deal is published, it needs to be stored, sealed, and proven in order for the provider to be paid. However, these components are handled by the {{<sref storage_mining_subsystem>}}. So the final task for the Storage Market is to handoff to the Storage Mining Subsystem
```

Comment:
nit: Likely worth mentioning in a sentence or two the high level of micropayments -> data for fil structure up top, so this can make sense

Reply:
I feel this is well documented in the storage deal section itself and I'd like to just sref it.